app/models/program_session.rb

```diff
@@ -69,8 +69,9 @@ class ProgramSession < ApplicationRecord
   end
   scope :emails, -> { joins(:speakers).pluck(:speaker_email).uniq }
 
-  scope :workshops, -> { joins(:session_format).where(session_format: { name: 'Workshop' }) }
-  scope :regular_sessions, -> { joins(:session_format).where(session_format: { name: 'Regular Session' } ) }
+  scope :in_session_format, ->(session_format) do
+    joins(:session_format).where(session_format: session_format)
```

Comment:
You probably don't need the `joins` here anymore and can just do `where(session_format_id: session_format.id)`.